src/utils/api.js

```diff
@@ -4,16 +4,21 @@ const instance = axios.create({
     baseURL: "https://nc-news-api-tts7.onrender.com/api/",
 })
 
-export const getArticles = (topic) => {
+export const getArticles = (topic, filters) => {
+    const { sortBy, order } = filters
+    let baseURL = `/articles?`
+
     if (topic) {
-        return instance.get(`/articles?topic=${topic}`).then(({ data }) => {
-            return data.articles
-        })
+        baseURL += `topic=${topic}&`
+    }
+    if (sortBy) {
+        baseURL += `sort_by=${sortBy}&`
+    }
+    if (order) {
+        baseURL += `order=${order}`
     }
 
-    return instance.get("/articles").then(({ data }) => {
-        return data.articles
-    })
+    return instance.get(baseURL).then(({ data }) => data.articles)
```

Comment:
Nice extending of the `getArticles` function Jess 👍 
You may find it useful to use the Axios params object here, it can do a lot of the functionality you've manually added (checking if a value exists and adding it to the query if it does), though obviously what you have already is working great.